### ISSUE TITLE
Integrate with Google's Cloud Test Lab Robo

### ISSUE THREAD

**Issue by bootstraponline** [google/Accessibility-Test-Framework-for-Android#3]
Google's Cloud Test Lab offers a Robo test which visits all views in an app. It'd be perfect to integrate the accessibility checks with this tool.

https://developers.google.com/cloud-test-lab/robo-ux-test

**Comment by caseyburkhardt** [comment]
Thanks for the feature request, will share with the team.  We've been working to bring our framework to the Play Store Pre-Launch report, which uses Cloud Test Lab infrastructure.  You can find accessibility reports within the Google Play Console > Release management > Pre launch report > Accessibility

[More info](https://support.google.com/googleplay/android-developer/answer/7002270?hl=en)

Most APKs uploaded since July 2018 should have accessibility results within their reports.
